ckanext/dcat/harvesters/_json.py

```diff
@@ -353,3 +354,11 @@ def copy_across_dataset_groups(existing_dataset, harvested_dataset):
 
     if existing_groups:
         harvested_dataset['groups'] = existing_groups
+
+def copy_across_dataset_tayside_elements(existing_dataset, harvested_dataset):
+    '''Copies the tayside elements any existing datasets
+    have to the newly harvested datasets'''
+    existing_frequency = existing_dataset.get('frequency')
+
+    if existing_frequency:
+        harvested_dataset.update({'frequency':existing_frequency})
```

Comment:
Why only frequency? If you do the opposite `existing_dataset.update(harvested_dataset)` think it will merge so that it keeps whatever there was in `existing_dataset` and overwrite everything from `harvested_dataset` including frequency. WDYT?

Reply:
```
>>> a = {'one': 1, 'two': 2, 'three': 3}
>>> 
>>> b = {'three': 33, 'four': 4, 'five': 5}
>>> a.update(b)
>>> a
{'one': 1, 'two': 2, 'three': 33, 'four': 4, 'five': 5}
```

Reply:
Not sure about that
If we implement something like that the reharvested dataset will always be using values of the existing dataset but what if there are some changes made in harvest dataset source and then reharvested. Now the changes won't be shown in the newly harvested dataset as it would still use previous values (from existing dataset).

Reply:
@MuhammadIsmailShahzad  We should do the opposite 

`existing_dataset.update(harvested_dataset)` - this way it will take everything from the harvested and override what exists or add if not exists (like in the example). + this way everything that was in `existing_dataset` but not in `harvested_dataset` will still remain. 

So you are updating existing dataset with a new one - not vice versa

Reply:
ok i get it now. Let me make the changes locally and will push after testing(manually).

Reply:
Implemented